modules/life-cycle-limited-support.adoc

```diff
@@ -5,14 +5,14 @@
 [id="rosa-limited-support_{context}"]
 = Limited support status
 
-When a cluster transitions to a _Limited Support_ status, it means that the SLA is no longer applicable and credits requested against the SLA are denied. It does not mean that you no longer have product support. You can return the cluster to a fully-supported status by remediating the violating factors.
+When a cluster transitions to a _Limited Support_ status, Red Hat no longer proactively monitors the cluster, the SLA is no longer applicable, and credits requested against the SLA are denied. It does not mean that you no longer have product support. In some cases, the cluster can return to a fully-supported status if you remediate the violating factors. However, in other cases, you might have to delete and recreate the cluster.
```

Comment:
```suggestion
When a cluster transitions to a _Limited Support_ status, Red Hat no longer proactively monitors the cluster, the SLA is no longer applicable, and credits requested against the SLA are denied. It does not mean that you no longer have product support. In some cases, the cluster can return to a fully-supported status if you remediate the violating factors. However, in other cases, you may have to delete and recreate the cluster.
```

Reply:
According to the IBM Style Guide: Use (may) only to indicate permission. Otherwise, use "can" or "might."

Reply:
@mburke5678: Isn't the use of 'might' correct over here, because the intent is to indicate probability or possibility?

Reply:
Yes.